setup.py

```diff
@@ -12,12 +12,33 @@
 needs_pytest = {'pytest', 'test', 'ptr'}.intersection(sys.argv)
 pytest_runner = ['pytest-runner'] if needs_pytest else []
 
+# Check Python version
+if sys.version_info[:2] < (3, 6):
+    print('This package requires a verion of Python>=3.6.'
+          'Please upgrading your version of Python to use MembraneCurvature.')
+    sys.exit(-1)
```

Comment:
might be a little more conventional to `raise RuntimeError(message)`

Reply:
I suspect @ojeda-e implemented this because it's the way we do it in MDAnalysis core. That being said, we probably should change that too (I just saw some py2 stuff in there too that needs to get deleted =.=).

Reply:
Yes, you're correct @IAlibay. I will follow @tylerjereddy suggestion here.